examples/pod_namespace_watch.py

```diff
@@ -31,6 +31,25 @@ def main():
     v1 = client.CoreV1Api()
     count = 10
     w = watch.Watch()
+    """
+    [INFO: `server-side` & `client-side` timeout]
+
+    `kwargs['timeout_seconds'] = n`:
+       - It is a server-side timeout, i.e. a polite request to the server,
+         mentioning it to cleanly close the connection after n seconds.
```

Comment:
https://github.com/kubernetes/apiserver/blob/92392ef22153d75b3645b0ae339f89c12767fb52/pkg/endpoints/handlers/get.go#L255

if it is not set, the default is a random timeout between 0 and minRequestTimeout
https://github.com/kubernetes/apiserver/blob/92392ef22153d75b3645b0ae339f89c12767fb52/pkg/endpoints/handlers/get.go#L258